projects/libfmt/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://github.com/pauldreik/fmt"
+primary_contact: "pauldreikossfuzz@gmail.com"
+auto_ccs:
+ - "victor.zverovich@gmail.com"
+
```

Comment:
nit: let's get rid of an empty newline.

Reply:
No worries, will fix that in the next pull request. Perhaps stuff like this could be checked by some tool (one runs the lengthy docker build/check scripts anyway, so adding an extra step would not disturb the workflow). Perhaps it could check that the addresses are valid, no trailing spaces etc.